lib/iris/fileformats/pp_save_rules.py

```diff
@@ -422,7 +422,13 @@ def _grid_and_pole_rules(cube, pp):
     lat_coord = vector_coord(cube, "latitude")
     grid_lat_coord = vector_coord(cube, "grid_latitude")
 
-    if lon_coord and not is_regular(lon_coord):
+    scalar_lon_coord = scalar_coord(cube, "longitude")
+
+    if lon_coord is None and grid_lon_coord is None and scalar_lon_coord:
+        pp.bdx = (unit := scalar_lon_coord.units) and unit.modulus or 360.0
```

Comment:
Is it `360.0` (or equivalent) because a 1-wide lon coord must wrap all the way round the globe?

Reply:
> Is it `360.0` (or equivalent) because a 1-wide lon coord must wrap all the way round the globe?

The default magical value of `360.0` if purely from the example PP fields that I've seen, and I guess makes sense given the definition of `bdx`. It seems to be the default that the model is outputting by example, otherwise I've not got any other evidence of what it should be.

Reply:
Please could you add a code comment to this effect?

Reply:
Done, see https://github.com/SciTools/iris/pull/5058/commits/311a5e3da49e665c410191dcb27f9a3106295827 :+1: